stat/stat.go

```diff
@@ -1293,3 +1294,76 @@ func MeanVariance(x, weights []float64) (mean, variance float64) {
 	variance = (ss - compensation*compensation/sumWeights) / (sumWeights - 1)
 	return mean, variance
 }
+
+// NormalSkewTest test whether the skew is different from the normal distribution.
+// This function tests the null hypothesis that the skewness of population that the
+// sample was drawn from is the same as that of a corresponding normal distribution.
+// Implementation based on scipy's: https://github.com/scipy/scipy/blob/v1.3.0/scipy/stats/stats.py#L1481-L1549
```

Comment:
Implementation details should not be in user-facing docs unless it is important for them to understand this.